server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,35 @@
+package psql
+
+import (
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+//AllMaintenanceRequests creates and returns a slice of datatype MaintenanceRequest with possible errors
+func (db *DB) AllMaintenanceRequests() ([]datastore.MaintenanceRequest, error) {
+	var maintenanceRequestTable []datastore.MaintenanceRequest
+	rows, err := db.connection.Query("SELECT * FROM maintenance_requests")
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		// Create a new MaintenanceRequest
+		var maintenance_request datastore.MaintenanceRequest
```

Comment:
could you change this to not use underscores, and do something like `maintenanceRequest` instead